tools/operator/launcher/k8s/health.go

```diff
@@ -53,11 +54,11 @@ func (k K8s) VerifyContainersAreRunning() error {
 	}()
 }
 
-func (k K8s) getReasonsPodsNotRunning(containers []string) error {
+func (k K8s) getReasonsPodsNotRunning(ns string, containers []string) error {
 
 	var reasonsPodsNotRunning []string
 	for i := 0; i < len(containers); i++ {
-		k.Arguments = []string{"describe", "pod", containers[i]}
+		k.Arguments = []string{"-n", ns, "describe", "pod", containers[i]}
```

Comment:
Same (replace with call to the SDK), this includes anywhere else in the K8s launcher we are using the CLI

Reply:
replaced with kubernetes sdk call